tools/scripts/README.md

```diff
@@ -30,6 +30,11 @@ be registered in the determined repo. Here is the process:
    artifact's `old` value to the previous `new` value, and set the new `new`
    value to the artifact produced by CI.
 
-5. Run `./bumpenvs.py bumpenvs.yaml`.  This will do a simple string replacement
+5. (optional) Run `./refresh-ubuntu-amis.py bumpenvs.yaml`.  This will fetch
```

Comment:
just to make sure, agent's ami are handled as part of step 4?

Reply:
Yeah, the agent amis are custom things built by the `environments repo`, but the master and bastion amis are totally default ubuntu images, and AFAICT have never been updated since we first wrote these templates.